m33ksauce/oralbibleapp
# Release 1.0.0

## What?

Includes the changes required to get our build ready for release. Should only include minor cleanup, but occasionally may include additional features if needed for release.

So far this includes:

- Upgrades to cordova
- Addition of a logo
- Styling for empty categories

## Why?

We are preparing to release v1.0.0 to production. These changes are needed for final polishing.

## How?

- Upgrade npm packages
- Add icon to assets and reference in main page
- Add a style to grey out categories that contain no children

## Testing?

- Start the app
- Verify Old Testament is a lighter-grey than the categories with children
- Verify main page has the logo
- Run `npm run package` and ensure the apk generates
- Run `npm run package-prod` and ensure the aab generates

## Screenshots (optional)

N/A

## Anything Else?

Nope
